src/main/java/seedu/address/model/delivery/DeliveryStatus.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.model.delivery;
+
+/**
+ * Represents the status of a delivery.
+ */
+public enum DeliveryStatus {
+    PENDING,
```

Comment:
Reminder to update the status into the four new groups : ORDERED, SHIPPED, COMPLETED, CANCELLED